superduperdb/container/model.py

```diff
@@ -91,7 +91,10 @@ def create_predict_job(
             },
         )
 
-    def _predict_one(self, X: int, **kwargs) -> int:
+    async def apredict_one(self, X: t.Any, **kwargs):
+        raise NotImplementedError
+
+    def predict_one(self, X: t.Any, **kwargs) -> int:
```

Comment:
(_Just double-checking that you meant to rename this from `_predict_one` to `predict_one`. No action needed if you did._)

Reply:
Yes

Reply:
I reverted this back to the form it had before. I also added not just `_apredict_one` but also `_apredicta